packages/vscode-extension/package.nls.json

```diff
@@ -422,9 +431,12 @@
+    "teamstoolkit.viewsWelcome.teamsfx-empty-project-with-chat.content": "Jump right into Teams Toolkit and get an overview of the fundamentals. For more information, visit [Get Started](command:fx-extension.openWelcome?%5B%22SideBar%22%5D).\nWalk through the steps to build a real-world Teams app.\n[Documentation](command:fx-extension.openDocument?%5B%22SideBar%22%5D)\n[How-to Guides](command:fx-extension.selectTutorials?%5B%22SideBar%22%5D)\nCreate a project or explore our samples.\n[Create a New App](command:fx-extension.create?%5B%22SideBar%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22SideBar%22%5D)\nStart with Github Copilot.\n[Start with Github Copilot](command:fx-extension.invokeChat?%5B%22SideBar%22%5D)",
+    "teamstoolkit.viewsWelcome.teamsfx-empty-project-new-user-with-chat.content": "Jump right into Teams Toolkit and get an overview of the fundamentals.\n[Get Started](command:fx-extension.openWelcome?%5B%22SideBar%22%5D)\nCreate a project or explore our samples.\n[Create a New App](command:fx-extension.create?%5B%22SideBar%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22SideBar%22%5D)\nWalk through the steps to build a real-world Teams app.\n[Documentation](command:fx-extension.openDocument?%5B%22SideBar%22%5D)\n[How-to Guides](command:fx-extension.selectTutorials?%5B%22SideBar%22%5D)\nStart with Github Copilot.\n[Start with Github Copilot](command:fx-extension.invokeChat?%5B%22SideBar%22%5D)",
     "teamstoolkit.viewsWelcome.teamsfx-feedback.content": "Take 2 minutes to help us improve, your feedback matters!\n[We Would Love Your Feedback](command:fx-extension.openSurvey)",
     "teamstoolkit.walkthroughs.description": "Jumpstart your Teams app development experience",
     "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.description": "Start building your first app with [Teams](https://aka.ms/teamsfx-capabilities-overview) or [Outlook add-in](https://aka.ms/teamsfx-outlook-add-in-capabilities) capabilities. Create it from scratch or explore our samples for real-world examples and code structures.\n[Create a New App](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22WalkThrough%22%5D)",
+    "teamstoolkit.walkthroughs.steps.teamsToolkitBuildAppWithChat.description": "Start building your first app with [Teams](https://aka.ms/teamsfx-capabilities-overview) or [Outlook add-in](https://aka.ms/teamsfx-outlook-add-in-capabilities) capabilities. Create it from scratch or explore our samples for real-world examples and code structures.\nYou can also start your Teams extension experience with the Github Copilot.\n[Create a New App](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%5D)\n[View Samples](command:fx-extension.openSamples?%5B%22WalkThrough%22%5D)\n [Start with Github Copilot](command:fx-extension.invokeChat?%5B%22WalkThrough%22%5D)\n__Tip: You need to have a Github subscription to use Github Copilot.__",
```

Comment:
Change "You can also start your Teams extension experience with the Github Copilot." to "Enhance your Teams extension experience with GitHub Copilot."

Reply:
Updated all strings. @supkasar could you help sign off?